neard/src/migrations.rs

```diff
@@ -272,15 +282,29 @@ lazy_static_include::lazy_static_include_bytes! {
     MAINNET_STORAGE_USAGE_DELTA => "res/storage_usage_delta.json",
 }
 
+#[cfg(feature = "protocol_feature_restore_receipts_after_fix")]
+lazy_static_include::lazy_static_include_bytes! {
```

Comment:
Maybe swap this with next constant so that things related to single migration is consecutive